### PR TITLE
:arrow_up: Update composer.lock for friendsofsymfony/user-bundle

### PR DESCRIPTION
**Pull request by alexislefebvre**
## Type
Feature

## Purpose
This PR update the `composer.lock` file in order to sync it with `composer.json`.

Remove "expired" from fixtures, see:
https://github.com/FriendsOfSymfony/FOSUserBundle/blob/master/Changelog.md#200-beta1-2016-11-29

Remove "locked" from fixtures, see:
FriendsOfSymfony/FOSUserBundle@7ad2a4b#diff-a1821a37b4b2e9240f70068f6bdea09e

## BC Break
NO

### CODE REVIEW

File: composer.lock [hunk 1]
```diff
@@ -448,7 +447,7 @@
             "keywords": [
                 "database"
             ],
-            "time": "2015-03-30 12:14:13"
+            "time": "2015-03-30T12:14:13+00:00"
```
Comment (alexislefebvre) [inline composer.lock]: This change in the date format is due to the upgrade of Composer to the 1.3 version.

**Comment by alexislefebvre** [comment]
There is an error due to this line: https://github.com/Victoire/victoire/blob/648350a6fe3ced8e615528ef4af14b91b0c1ce54/Tests/Functionnal/app/AppKernel.php#L136

It looks like the code comes from here: https://github.com/symfony/symfony/issues/11236#issuecomment-199666629

**Comment by alexislefebvre** [comment]
I removed the `cleanupContainer` method from `AppKernel`, because of this error:

    ReflectionProperty->setValue() at /home/alexis/Works/troopers/victoire/victoire-fork/Tests/Functionnal/app/AppKernel.php:137

For reference, here is the full error log:

```
vendor/bin/behat -vv Tests/Features/Blog/create.feature:8
@mink:selenium2 @alice(Page) @reset-schema
Feature: Create a blog

  Background:                   # Tests/Features/Blog/create.feature:4
    Given I maximize the window # Victoire\Tests\Features\Context\JavascriptContext::iMaximizeTheBroswer()
    And I am on homepage        # Knp\FriendlyContexts\Context\MinkContext::iAmOnHomepage()

  Scenario: I create a new blog                                      # Tests/Features/Blog/create.feature:8
    Given I open the hamburger menu                                  # Victoire\Tests\Features\Context\VictoireContext::iOpenTheHamburgerMenu()
      Element not found in the page after 10 seconds"
    Then I should see "Blog"                                         # Knp\FriendlyContexts\Context\MinkContext::assertPageContainsText()
    When I follow "Blog"                                             # Knp\FriendlyContexts\Context\MinkContext::clickLink()
    Then I should see "Aucun résultat"                               # Knp\FriendlyContexts\Context\MinkContext::assertPageContainsText()
    When I follow "Créer un nouveau blog"                            # Knp\FriendlyContexts\Context\MinkContext::clickLink()
    Then I should see "Nom"                                          # Knp\FriendlyContexts\Context\MinkContext::assertPageContainsText()
    When I fill in "Nom" with "The Jedi network"                     # Knp\FriendlyContexts\Context\MinkContext::fillField()
    And I follow "Créer"                                             # Knp\FriendlyContexts\Context\MinkContext::clickLink()
    Then I should see "Page créée avec succès"                       # Knp\FriendlyContexts\Context\MinkContext::assertPageContainsText()
    And I switch to "layout" mode                                    # Victoire\Tests\Features\Context\VictoireContext::iSwitchToMode()
    And I should see "Nouveau contenu"                               # Knp\FriendlyContexts\Context\MinkContext::assertPageContainsText()
    When I select "Force" from the "1" select of "main_content" slot # Victoire\Tests\Features\Context\VictoireContext::iSelectFromTheSelectOfSlot()
    Then I should see "Créer"                                        # Knp\FriendlyContexts\Context\MinkContext::assertPageContainsText()
    When I fill in "Côté de la force" with "obscur"                  # Knp\FriendlyContexts\Context\MinkContext::fillField()
    And I submit the widget                                          # Victoire\Tests\Features\Context\VictoireContext::iSubmitTheWidget()


                                                                              
  [Symfony\Component\Debug\Exception\ContextErrorException]                   
  Warning: Cannot bind closure to scope of internal class ReflectionProperty  
                                                                              

Exception trace:
 () at /tmp/Victoire/cache/ci/appCiDebugProjectContainer.php:11132
 Symfony\Component\Debug\ErrorHandler->handleError() at n/a:n/a
 Closure->bindTo() at /tmp/Victoire/cache/ci/appCiDebugProjectContainer.php:11132
 FOSUserBundleDoctrineUserManager_000000005a1b138c000000005bf63e67f721f3e7ab9b7fa27d1bafa8f8a8cec1->__set() at n/a:n/a
 ReflectionProperty->setValue() at /home/alexis/Works/troopers/victoire/victoire-fork/Tests/Functionnal/app/AppKernel.php:137
 AppKernel->cleanupContainer() at /home/alexis/Works/troopers/victoire/victoire-fork/Tests/Functionnal/app/AppKernel.php:111
 AppKernel->shutdown() at /home/alexis/Works/troopers/victoire/victoire-fork/vendor/behat/symfony2-extension/src/Behat/Symfony2Extension/Context/Initializer/KernelAwareInitializer.php:70
 Behat\Symfony2Extension\Context\Initializer\KernelAwareInitializer->rebootKernel() at n/a:n/a
 call_user_func() at /home/alexis/Works/troopers/victoire/victoire-fork/vendor/symfony/symfony/src/Symfony/Component/EventDispatcher/EventDispatcher.php:184
 Symfony\Component\EventDispatcher\EventDispatcher->doDispatch() at /home/alexis/Works/troopers/victoire/victoire-fork/vendor/behat/behat/src/Behat/Testwork/EventDispatcher/TestworkEventDispatcher.php:39
 Behat\Testwork\EventDispatcher\TestworkEventDispatcher->dispatch() at /home/alexis/Works/troopers/victoire/victoire-fork/vendor/behat/behat/src/Behat/Behat/EventDispatcher/Tester/EventDispatchingScenarioTester.php:117
 Behat\Behat\EventDispatcher\Tester\EventDispatchingScenarioTester->tearDown() at /home/alexis/Works/troopers/victoire/victoire-fork/vendor/behat/behat/src/Behat/Behat/Tester/Runtime/IsolatingScenarioTester.php:70
 Behat\Behat\Tester\Runtime\IsolatingScenarioTester->test() at /home/alexis/Works/troopers/victoire/victoire-fork/vendor/behat/behat/src/Behat/Behat/Tester/Runtime/RuntimeFeatureTester.php:84
 Behat\Behat\Tester\Runtime\RuntimeFeatureTester->test() at /home/alexis/Works/troopers/victoire/victoire-fork/vendor/behat/behat/src/Behat/Behat/Hook/Tester/HookableFeatureTester.php:72
 Behat\Behat\Hook\Tester\HookableFeatureTester->test() at /home/alexis/Works/troopers/victoire/victoire-fork/vendor/behat/behat/src/Behat/Behat/EventDispatcher/Tester/EventDispatchingFeatureTester.php:71
 Behat\Behat\EventDispatcher\Tester\EventDispatchingFeatureTester->test() at /home/alexis/Works/troopers/victoire/victoire-fork/vendor/behat/behat/src/Behat/Testwork/Tester/Runtime/RuntimeSuiteTester.php:63
 Behat\Testwork\Tester\Runtime\RuntimeSuiteTester->test() at /home/alexis/Works/troopers/victoire/victoire-fork/vendor/behat/behat/src/Behat/Testwork/Hook/Tester/HookableSuiteTester.php:73
 Behat\Testwork\Hook\Tester\HookableSuiteTester->test() at /home/alexis/Works/troopers/victoire/victoire-fork/vendor/behat/behat/src/Behat/Testwork/EventDispatcher/Tester/EventDispatchingSuiteTester.php:72
 Behat\Testwork\EventDispatcher\Tester\EventDispatchingSuiteTester->test() at /home/alexis/Works/troopers/victoire/victoire-fork/vendor/behat/behat/src/Behat/Testwork/Tester/Runtime/RuntimeExercise.php:71
 Behat\Testwork\Tester\Runtime\RuntimeExercise->test() at /home/alexis/Works/troopers/victoire/victoire-fork/vendor/behat/behat/src/Behat/Testwork/EventDispatcher/Tester/EventDispatchingExercise.php:70
 Behat\Testwork\EventDispatcher\Tester\EventDispatchingExercise->test() at /home/alexis/Works/troopers/victoire/victoire-fork/vendor/behat/behat/src/Behat/Testwork/Ordering/OrderedExercise.php:80
 Behat\Testwork\Ordering\OrderedExercise->test() at /home/alexis/Works/troopers/victoire/victoire-fork/vendor/behat/behat/src/Behat/Testwork/Tester/Cli/ExerciseController.php:149
 Behat\Testwork\Tester\Cli\ExerciseController->testSpecifications() at /home/alexis/Works/troopers/victoire/victoire-fork/vendor/behat/behat/src/Behat/Testwork/Tester/Cli/ExerciseController.php:108
 Behat\Testwork\Tester\Cli\ExerciseController->execute() at /home/alexis/Works/troopers/victoire/victoire-fork/vendor/behat/behat/src/Behat/Testwork/Cli/Command.php:63
 Behat\Testwork\Cli\Command->execute() at /home/alexis/Works/troopers/victoire/victoire-fork/vendor/symfony/symfony/src/Symfony/Component/Console/Command/Command.php:259
 Symfony\Component\Console\Command\Command->run() at /home/alexis/Works/troopers/victoire/victoire-fork/vendor/symfony/symfony/src/Symfony/Component/Console/Application.php:849
 Symfony\Component\Console\Application->doRunCommand() at /home/alexis/Works/troopers/victoire/victoire-fork/vendor/symfony/symfony/src/Symfony/Component/Console/Application.php:193
 Symfony\Component\Console\Application->doRun() at /home/alexis/Works/troopers/victoire/victoire-fork/vendor/behat/behat/src/Behat/Testwork/Cli/Application.php:122
 Behat\Testwork\Cli\Application->doRun() at /home/alexis/Works/troopers/victoire/victoire-fork/vendor/symfony/symfony/src/Symfony/Component/Console/Application.php:124
 Symfony\Component\Console\Application->run() at /home/alexis/Works/troopers/victoire/victoire-fork/vendor/behat/behat/bin/behat:34

behat [-s|--suite SUITE] [-f|--format FORMAT] [-o|--out OUT] [--format-settings FORMAT-SETTINGS] [--init] [--lang LANG] [--name NAME] [--tags TAGS] [--role ROLE] [--story-syntax] [-d|--definitions DEFINITIONS] [--snippets-for [SNIPPETS-FOR]] [--snippets-type SNIPPETS-TYPE] [--append-snippets] [--no-snippets] [--strict] [--order ORDER] [--rerun] [--stop-on-failure] [--dry-run] [--] [<paths>]
```

**Comment by paulandrieux** [comment]
Dont you think the cleanupContainer refactor should be in a distinct PR ?

**Comment by alexislefebvre** [comment]
It requires #787.

**Comment by alexislefebvre** [comment]
@lenybernard I removed the `composer.lock` on another branch, here is a PR to ensure that it won't break CircleCI : https://github.com/Victoire/victoire/pull/788 The first build failed: https://circleci.com/gh/alexislefebvre/victoire/39

**Pull request closed without merging**